modules/tinymce/src/core/test/ts/browser/html/SchemaTest.ts

```diff
@@ -490,4 +490,32 @@ describe('browser.tinymce.core.html.SchemaTest', () => {
       }
     });
   });
+
+  context('custom elements', () => {
+    it('TBA: custom elements are added as element rules and copy the span/div rules', () => {
```

Comment:
It's not related to this change and I just added it as I realised there were no tests for custom elements at all. So as we don't know when that was added (and it would have been before Jira) we don't have a number we can use here.